app/providers/genericGitHubDataService.ts

```diff
@@ -1,27 +1,48 @@
+import {Storage, LocalStorage} from "ionic-angular";
 import {Injectable} from "@angular/core";
 import {Http, HTTP_PROVIDERS, Response, Headers, RequestOptions} from "@angular/http";
 import "rxjs/Rx";
 import {GitHubFileItem} from "../model/gitHub/gitHubFileItem";
 import {GitHubFileContentItem} from "../model/gitHub/gitHubFileContentItem";
+import {AuthService} from "./authService";
```

Comment:
Remove ?, since AuthService is not being used this file ... token is the only contract